autoware_map_msgs/msg/PointCloudMapCellWithID.msg

```diff
@@ -0,0 +1,5 @@
+# Pointcloud data with ID
+string cell_id
+sensor_msgs/PointCloud2 pointcloud
+geometry_msgs/Point min_point
+geometry_msgs/Point max_point
```

Comment:
Are these the boundaries of the point cloud or the cell? (Blue or the red from the image below?)

![Untitled](https://user-images.githubusercontent.com/10751153/195024377-ea0825db-92c2-4ee7-be55-58c3b0f747da.png)

Reply:
I meant the blue one. But now I am not sure whether we should add this boundary information or not.

For the blue type of boundary: Speaking of my dynamic map loading implementation, it is nice to have it in the message, but not necessary (without this boundary info, we should compute min-max again inside `ndt_scan_matcher`, but this is not a big problem for now).

For the red type of boundary: I don't think we need it. It is not useful for `ndt_scan_matcher` at least.

Which type of boundary did you have in mind? @xmfcx

Reply:
@kminoda I think red makes more sense because it determines the boundaries of the cell.

I don't know if the information about the blue one is used anywhere.

Truth is, since we are making use of a predetermined grid, it should be just enough to share 
- the size of a cell (in meters)
- integer coordinates (origin being 0,0)

This way we wouldn't need an id, we would just share coordinates.

And we would know exactly what cell we are sharing in both the client and the server.

If the `ndt_scan_matcher` uses blue boundary, we can also compute and share it too.

Reply:
But again, if I am complicating this PR too much, we can just add it the way you intended, and we can think about changing the implementation later on. So please tell if you think it is better to just add it in a way you like.

Reply:
No worries for that. Thanks again for your idea!

> Truth is, since we are making use of a predetermined grid, it should be just enough to share...

I agree that these data is informative enough, and may be necessary if the client wants to have a prior knowledge about the map in order to implement a smarter logic for loading/unloading the map cells. But as far as we've discussed in TIER IV, we could not find any concrete use case in which such boundary information is necessary for the client side.
(FYI: [the proposal A shown in the previous discussion](https://github.com/orgs/autowarefoundation/discussions/2812) also communicates the similar data.)

So my proposal for this comment would be:
- Leave boundary information out of `PointCloudMapCellsWithID`
- Discuss whether to include the boundary information if the needs arise in the future

Reply:
Ok, let's leave it out then.

Reply:
@xmfcx 
Sorry I forgot to reflect the conclusion.
Here's the commit: https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/7dd600f7ee8f2c91a6670f56d648b871ab3d1651